src/JobPanel/backend/connection.py

```diff
@@ -443,7 +445,8 @@ def get_delegator(self, local_service):
                     connected = True
                     break
             if connected:
-                self._delegator_proxy = ServiceProxy(local_service._repeater, {}, self)
+                from .service_proxy import ServiceProxy
```

Comment:
OK, napis ty importy uvnir kodu do komentare na zacatku souboru a budeme to resit pozdeji, az to bude nejak fungovat.